### PR TITLE
Sets up coveralls

### PR DESCRIPTION
**Pull request by HannaSwDn**
## PT:
[Link to PT](https://www.pivotaltracker.com/story/show/162407729)

## Actions taken:
Adds coveralls for test coverage report

## Lessons learned:
The meaning of coveralls and how to set it up

### CODE REVIEW

**Review by schneiderman805 — COMMENTED** [review]
I have reviewed it and it looks good, had to do it on my phone. If you would like to go over it with me later tonight that’s cool.

**Review by shahin1290 — APPROVED** [review]
It look fine to me

File: coverage/.resultset.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (oliverochman) [inline coverage/.resultset.json]: you need to add `coverage/*` to the .gitignore

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: coverage/.resultset.json (hunk 1 above)
Comment (HannaSwDn) [reply coverage/.resultset.json]: Done

**Pull request merged**